backend/src/main/java/com/google/sps/data/MatchPreference.java

```diff
@@ -41,4 +41,18 @@ public static MatchPreference forIntValue(int value) {
     }
     throw new IllegalStateException("Unknown enum value.");
   }
+
+  /** Parse match preference string to MatchPreference */
+  public static MatchPreference forStringValue(String value) {
+    switch (value.toLowerCase()) {
+      case "different":
+        return MatchPreference.DIFFERENT;
+      case "any":
+        return MatchPreference.ANY;
+      case "similar":
+        return MatchPreference.SIMILAR;
+      default:
+        throw new IllegalStateException("Unknown enum value.");
+    }
+  }
```

Comment:
This was submitted already in another PR right?

Reply:
Yes but I needed it for this PR so I just added it so it would build.